src/Storage/ConsentStorage.php

```diff
@@ -3,11 +3,10 @@
 namespace OpenIDConnectServer\Storage;
 
 const STICKY_CONSENT_DURATION = 7 * DAY_IN_SECONDS;
-const META_KEY_PREFIX         = 'oidc_consent_timestamp';
 
 class ConsentStorage {
 	private function get_meta_key( $client_id ) : string {
-		return META_KEY_PREFIX . '_' . $client_id;
+		return 'oidc_' . $client_id . '_consent_timestamp';
```

Comment:
Do we really need this change?

Reply:
It was only to be consistent with all fields being namespaced like `oidc_clientid_`. Can be reverted now that we are not doing it.

Reply:
Reverted!